pkgs/os-specific/linux/alsa-project/alsa-firmware/default.nix

```diff
@@ -1,20 +1,19 @@
-{ lib, stdenv, buildPackages, autoreconfHook, fetchurl, fetchpatch }:
+{ lib, stdenv, autoreconfHook, fetchurl }:
 
 stdenv.mkDerivation rec {
   pname = "alsa-firmware";
-  version = "1.2.1";
+  version = "1.2.4";
 
   src = fetchurl {
     url = "mirror://alsa/firmware/alsa-firmware-${version}.tar.bz2";
-    sha256 = "1aq8z8ajpjvcx7bwhwp36bh5idzximyn77ygk3ifs0my3mbpr8mf";
+    sha256 = "sha256-tnttfQi8/CR+9v8KuIqZwYgwWjz1euLf0LzZpbNs1bs=";
   };
 
-  patches = [ (fetchpatch {
-    url = "https://github.com/alsa-project/alsa-firmware/commit/a8a478485a999ff9e4a8d8098107d3b946b70288.patch";
-    sha256 = "0zd7vrgz00hn02va5bkv7qj2395a1rl6f8jq1mwbryxs7hiysb78";
-  }) ];
+  postPatch = ''
+    export CC_FOR_BUILD=$CC
```

Comment:
This likely breaks cross compilation if it actually ever uses `CC_FOR_BUILD` for something (which is the only way I could imagine this working now). Try instead:

```
  depsBuildBuild = [ buildPackages.stdenv.cc ];
```

which sets `CC_FOR_BUILD` already.

Reply:
This seems to have fixed it, thanks!